RestSharp/Authenticators/OAuth/OAuthWorkflow.cs

```diff
@@ -250,169 +230,130 @@ public virtual OAuthWebQueryInfo BuildClientAuthAccessTokenInfo(string method, W
                         break;
                 }
-            }
-
-            string timestamp = OAuthTools.GetTimestamp();
-            string nonce = OAuthTools.GetNonce();
-
-            this.AddAuthParameters(parameters, timestamp, nonce);
-
-            string signatureBase = OAuthTools.ConcatenateRequestElements(method, url, parameters);
-            string signature = OAuthTools.GetSignature(this.SignatureMethod, this.SignatureTreatment, signatureBase,
-                this.ConsumerSecret, this.TokenSecret);
-
-            OAuthWebQueryInfo info = new OAuthWebQueryInfo
-                                     {
-                                         WebMethod = method,
-                                         ParameterHandling = this.ParameterHandling,
-                                         ConsumerKey = this.ConsumerKey,
-                                         Token = this.Token,
-                                         SignatureMethod = this.SignatureMethod.ToRequestValue(),
-                                         SignatureTreatment = this.SignatureTreatment,
-                                         Signature = signature,
-                                         Timestamp = timestamp,
-                                         Nonce = nonce,
-                                         Version = this.Version ?? "1.0",
-                                         Callback = this.CallbackUrl,
-                                         ConsumerSecret = this.ConsumerSecret,
-                                         TokenSecret = this.TokenSecret
-                                     };
+
+            var timestamp = OAuthTools.GetTimestamp();
+            var nonce = OAuthTools.GetNonce();
+
+            AddAuthParameters(parameters, timestamp, nonce);
+
+            var signatureBase = OAuthTools.ConcatenateRequestElements(method, url, parameters);
+            var signature = OAuthTools.GetSignature(SignatureMethod, SignatureTreatment, signatureBase,
+                ConsumerSecret, TokenSecret);
+
+            var info = new OAuthWebQueryInfo
+            {
+                WebMethod = method,
+                ParameterHandling = ParameterHandling,
+                ConsumerKey = ConsumerKey,
+                Token = Token,
+                SignatureMethod = SignatureMethod.ToRequestValue(),
+                SignatureTreatment = SignatureTreatment,
+                Signature = signature,
+                Timestamp = timestamp,
+                Nonce = nonce,
+                Version = Version ?? "1.0",
+                Callback = CallbackUrl,
+                ConsumerSecret = ConsumerSecret,
+                TokenSecret = TokenSecret
+            };
 
             return info;
         }
 
         private void ValidateTokenRequestState()
         {
-            if (this.RequestTokenUrl.IsNullOrBlank())
-            {
+            if (RequestTokenUrl.IsNullOrBlank())
```

Comment:
Is it not a better coding convention to always have curly braces after an if? (for this entire class)